brainglobe_atlasapi/atlas_generation/validate_atlases.py

```diff
@@ -111,9 +111,29 @@ def validate_checksum(atlas: BrainGlobeAtlas):
     pass
 
 
-def check_additional_references(atlas: BrainGlobeAtlas):
-    # check additional references are different, but have same dimensions
-    pass
+def validate_image_dimensions(atlas: BrainGlobeAtlas):
+    """
+    Check that annotation and reference image have the same dimensions.
+    """
+    assert (
+        atlas.annotation.shape == atlas.reference.shape
+    ), "Annotation and reference image do not have the same dimensions."
```

Comment:
```suggestion
    if atlas.annotation.shape != atlas.reference.shape:
        raise ValueError(
            "Annotation and reference image do not have the same dimensions. \n"
            f"Annotation image has dimension: {atlas.annotation.shape}, "
            f"while reference image has dimension {atlas.reference.shape}."
        )
```

Reply:
In this way the error is clear and precise about what it wrong

Reply:
Thank you!